x-pack/plugins/observability/common/typings.ts

```diff
@@ -4,5 +4,9 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
+import * as t from 'io-ts';
 
 export type Maybe<T> = T | null | undefined;
+
+export const alertStatusRt = t.union([t.literal('all'), t.literal('open'), t.literal('closed')]);
```

Comment:
This isn't really in scope of this ticket _directly_, but I'm nervous that we're still using "open" and "closed" to mean "active" and "recovered" because they feel like very different concepts.

Reply:
The only place "active" and "recovered" is used is on the control itself. `kibana.rac.alert.status` can be "open" or "closed". They aren't different in the context of observability alerts. @katrin-freihofner do you think we should change the text in the toggle control to be "open"/"closed"?

Reply:
I think we want to keep active and recovered as the language here (imo), but that the concepts are different in the underlying data so we'll eventually not want them linked. We've talked about it in 3 or 4 places but these things are hard to keep track of. But there will eventually be a difference between workflow status and active/recovered, since one is controlled by humans and the other is controlled by data + conditions of the rule. 

Does the alert schema not have any other status fields?

Reply:
I think this is still accurate: https://github.com/elastic/kibana/tree/master/x-pack/plugins/rule_registry#schema

Reply:
Oh thanks. I think we may need to use alerts that have this value set:
```
kibana.rac.alert.end: the ISO timestamp of the time at which the alert recovered.
```
I just now realized we probably want a way to have the investigation window (date range) apply to recovery date and not just start date ... but that's for later :P

Reply:
Let's leave it for now and address it as it comes up, but it will be important not to forget because I'm not 100% sure we will set that status to closed on resolution ... or if the semantics of when it gets set to closed / open could change and be surprising to us later.